tests/library/playwright.config.ts

```diff
@@ -128,7 +128,13 @@ for (const browserName of browserNames) {
       metadata: {
         platform: process.platform,
         docker: !!process.env.INSIDE_DOCKER,
-        headful: !!headed,
+        headless: (() => {
```

Comment:
Are you sure this will be fine on the dashboard? AFAIK, it is a boolean today, so maybe it won't work?

Reply:
we'll see!